packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -1324,6 +1148,15 @@ export const useArbTokenBridge = (
     setExecutedMessagesCache({ ...executedMessagesCache, ...added })
   }
 
+  const setWithdrawalsInStore = (withdrawalTxns: L2ToL1EventResultPlus[]) => {
+    let pwMap = {} as PendingWithdrawalsMap
```

Comment:
I would use `set` or `map` here

Reply:
Good point. However, `PendingWithdrawalsMap` logic is an old implementation and will get removed over time. It is currently being used in a lot of business logic for updating/adding transactions in UI so I am super cautious to not change the data-structure as the impact might introduce even more diffs.